doc/getting-started/getting-started.rst

```diff
@@ -132,6 +132,7 @@ To set up the ACRN build environment on the development computer:
            libusb-1.0-0-dev \
            python3 \
            python3-pip \
+           python3.8-venv \
```

Comment:
@weichenli-intel Line 409 below has python3-venv. Do we need both python3-venv and python3.8-venv? If we only need one, which is better?

Reply:
python3.8-venv is better.